src/Administration/Resources/app/administration/src/module/sw-extension-sdk/page/sw-extension-sdk-module/index.js

```diff
@@ -32,6 +32,14 @@ Shopware.Component.register('sw-extension-sdk-module', {
         showSearchBar() {
             return this.module?.displaySearchBar ?? true;
         },
+
+        showLanguageSwitch() {
+            return !!this.module?.displayLanguageSwitch;
+        },
+
+        smartBarButtons() {
+            return Shopware.State.get('extensionSdkModules').smartBarButtons;
```

Comment:
You can all smart bar buttons from all activated apps. It would be better if the smartBarButtons would contain an information that it only gets shown at the correct places (e.g. only your app)

Reply:
Ahh I get what you meant by using locationId now. Thanks a lot 😄